WordPress/src/main/res/values/colors.xml

```diff
@@ -111,4 +111,7 @@
     <color name="grey_lighten_20">#c8d7e1</color>
     <color name="grey_300">#ffe0e0e0</color>
     <color name="grey_900">#ff212121</color>
+
+    <!-- Site creation  -->
+    <color name="site_creation_intent_item_emoji_bg">#dcdcdc</color>
```

Comment:
In the designs I think the value is `#E9EFF5`, which is a bit lighter shade of grey.
Should we update it?

Reply:
Sure :+1: . I made a "rough guess" for these values - do you know if the dark-mode value is correct? I didn't see these specified anywhere :thinking:

Reply:
I wonder whether these should be specified as standard variables :thinking: . Originally (locally), I chose standard colors:

Light: `@color/gray_5` == `#dcdcde`
Dark: `@color/gray_70` == `#3c434a`

But these weren't neutral (especially the dark color), which stood out against the container background with some significant chromatic contrast (i.e. the circle background appears blue-ish).

Container background light: `#ffffff`
Container background dark: `#2d2d2d`

Also, the next value available for lighter gray was `@color/gray_0` which is: `#f6f7f7`, which seemed too light. So, somewhat arbitrarily, I picked `#3d3d3d` for dark and a slightly more neutral `#dcdcdc` for light.

It would be good to get exact values, though. @kyleaparker do you have specific values for these in mind (for dark and light modes)?

Reply:
> do you know if the dark-mode value is correct? I didn't see these specified anywhere 🤔
Indeed, we don't have it specified but I we can go with a value that is somehow similar in terms of contrast compared to the background color on the list.

The value for the dark-mode looks good, it's of a similar contrast seen through my eyes 👀 :

<details>
<summary>Preview</summary>
<br>
<img width="314" src="https://user-images.githubusercontent.com/4588074/159436945-2763166a-47aa-446b-ba87-eac65e6a66cc.png">
</details>

We can ask @kyleaparker for feedback on the icon bg in dark mode (preview above), and adjust it afterwards; I wouldn't block the PR for the color though.